Training web/Task04-Pixel Perfect/scss/styles.scss

```diff
@@ -0,0 +1,322 @@
+*, *:before, *:after {
+    -webkit-box-sizing: border-box;
+       -moz-box-sizing: border-box;
+            box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    flex-flow: column wrap;
+    margin: 0;
+}
+
+
+.nav {
+    height: 77px;
+    background-color: black;
+    display: flex;
+    flex-flow: row;
+    justify-content: space-between;
+    &__logo {
+        margin: 13px 73px 3px 19px;
+    }
+    &__menu {
+        display: none;
+        margin: 0;
+
+        p {
```

Comment:
No usés tags en los estilos, la razón podés leerla [acá](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)... el HTML no debería definir el estilo de tu sitio, es poco escalable.